src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -95,9 +95,11 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote" Key="$(RemoteLanguageServiceKey)" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote.15.7" Key="$(RemoteLanguageServiceKey)" />
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Remote.15.8" Key="$(RemoteLanguageServiceKey)" />
```

Comment:
❓ Why do we still need the 15.7 and 15.8 assemblies listed here?

Reply:
LiveShare is a single VSIX that's still supported on 15.6 and above. So we still need these dlls.

Reply:
@srivatsn That's fine, but why do the Roslyn 16.0 assemblies need IVTs to LiveShare for 15.7, when LiveShare for 16.0 will never try to load LiveShare for 15.7?

Reply:
Good point. Removed the 15.7 and 15.8 entries.

Reply:
Just to call it out: _technically_ it's actually problematic to remove the IVTs in this branch, since it means anybody running code out of this branch in 15.8 or 15.9 _and_ tries to use LiveShare in the F5 scenario will now be broken. It's entirely safe to have only 16.0 in the -vs-deps branch, but in general this has to have as many in there as VS branches that this targets.

Now that we have the technically correct answer, this is probably fine. I doubt anybody needs that, and if we do we can fix it then.